databox/core/index.ts

```diff
@@ -1,84 +1,76 @@
-import { IDataboxApis } from '@ulixee/databox-interfaces/IDataboxApis';
-import IDataboxPackage from '@ulixee/databox-interfaces/IDataboxPackage';
-import ConnectionToClient from '@ulixee/net/lib/ConnectionToClient';
-import ITransportToClient from '@ulixee/net/interfaces/ITransportToClient';
+import * as Fs from 'fs';
+import { NodeVM, VMScript } from 'vm2';
+import DataboxWrapper from '@ulixee/databox';
 import IDataboxManifest from '@ulixee/databox-interfaces/IDataboxManifest';
 import IDataboxModuleRunner from './interfaces/IDataboxModuleRunner';
-import PackageRegistry from './lib/PackageRegistry';
-import env from './env';
+import ServerHooks from './lib/ServerHooks';
 
-type IDataboxConnectionToClient = ConnectionToClient<IDataboxApis, {}>;
+export default class DataboxCore implements IDataboxModuleRunner {
+  public runsDataboxModuleVersion = '';
+  public runsDataboxModule = '@ulixee/databox';
 
-export default class DataboxCore {
-  public static connections = new Set<IDataboxConnectionToClient>();
-  public static databoxesDir: string;
+  private compiledScriptsByPath = new Map<string, Promise<VMScript>>();
 
-  private static runnersByModuleName: { [module: string]: IDataboxModuleRunner } = {};
-  private static packageRegistry: PackageRegistry;
-  private static apiHandlers: IDataboxApis = {
-    'Databox.upload': DataboxCore.upload.bind(this),
-    'Databox.run': DataboxCore.run.bind(this),
-  };
+  private vm = new NodeVM({
+    console: 'inherit',
+    sandbox: {},
+    wasm: false,
+    eval: false,
+    wrapper: 'commonjs',
+    strict: true,
+    require: {
+      external: ['@ulixee/*'],
+    },
+  });
 
-  public static addConnection(
-    transport: ITransportToClient<IDataboxApis, {}>,
-  ): IDataboxConnectionToClient {
-    const connection = new ConnectionToClient(transport, this.apiHandlers);
-    connection.once('disconnected', () => this.connections.delete(connection));
-    this.connections.add(connection);
-    return connection;
+  public async start(): Promise<void> {
+    process.env.ULX_DATABOX_DISABLE_AUTORUN = 'Y';
```

Comment:
Ok. The documentation for this variable (or the former named one) said that only needed to be "truthy".

Reply:
Yeah, we have started in some places to read the boolean so you can turn it off too. It does seem like this would work. It just wouldn't work with an N (I'm guessing)

Reply:
I just pushed a commit that changes all the values to `true` + I updated the documentation to recommend setting it to `true` instead of truthy.